Ontology/Willow/Asset/Equipment/HVAC/Chiller/Chiller.json

```diff
@@ -30,6 +30,14 @@
       },
       "writable": true
     },
+    {
+      "@type": "Component",
+      "displayName": {
+        "en": "Evaporator"
```

Comment:
Lines 17 through 30 allow the chiller to be described as water cooled or air cooled. Should that be deleted with the new asset types for water cooled and air cooled chillers?

Reply:
Ideally, yes. But removing would introduce a breaking change. We would need to analyze existing twins to see how many have defined this property.

Reply:
Understood and agreed